frontend/src/metabase-lib/queries/structured/Aggregation.ts

```diff
@@ -157,9 +157,7 @@ export default class Aggregation extends MBQLClause {
       return this.aggregation().isValid();
     } else if (this.isStandard() && this.dimension()) {
       const dimension = this.dimension();
-      const aggregationOperator = this.query()
-        .table()
-        .aggregationOperator(this[0]);
+      const aggregationOperator = this.query().aggregationOperator(this[0]);
```

Comment:
after we click `Visualize`
<img width="424" alt="image" src="https://user-images.githubusercontent.com/1937582/225986179-7cad4e36-ccdb-41a2-81f2-8d85d09bad91.png">

MBQL validation will kick in. So this part needs to call query's `aggregationOperator` instead of `Table` since tables aren't aware of expressions.